cli/src/util/spm.ts

```diff
@@ -69,7 +69,7 @@ let package = Package(
             targets: ["CapApp-SPM"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", branch: "main")`;
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "6.0.0")`;
```

Comment:
maybe here we might want to set the version to the locally installed `@capacitor/ios` version?

Reply:
yeah, the fact that this is actually pulling in whatever the latest is from github versus what was installed via npm is problematic